dist/scribe-plugin-content-cleaner.js

```diff
@@ -1,27 +1,37 @@
 (function(f){if(typeof exports==="object"&&typeof module!=="undefined"){module.exports=f()}else if(typeof define==="function"&&define.amd){define([],f)}else{var g;if(typeof window!=="undefined"){g=window}else if(typeof global!=="undefined"){g=global}else if(typeof self!=="undefined"){g=self}else{g=this}g.scribePluginContentCleaner = f()}})(function(){var define,module,exports;return (function e(t,n,r){function s(o,u){if(!n[o]){if(!t[o]){var a=typeof require=="function"&&require;if(!u&&a)return a(o,!0);if(i)return i(o,!0);var f=new Error("Cannot find module '"+o+"'");throw f.code="MODULE_NOT_FOUND",f}var l=n[o]={exports:{}};t[o][0].call(l.exports,function(e){var n=t[o][1][e];return s(n?n:e)},l,l.exports,e,t,n,r)}return n[o].exports}var i=typeof require=="function"&&require;for(var o=0;o<r.length;o++)s(r[o]);return s})({1:[function(require,module,exports){
 "use strict";
 
-module.exports = function (config) {
+module.exports = function (scribe) {
+    "use strict";
 
-    var replacers = [function (text) {
-        text.replace(/\s+/g, " ");
+    var COMMAND_NAME = "cleanup";
+
+    var reducers = [function (text) {
+        return text.replace(/\s+/g, " ");
     }, function (text) {
-        text.replace("--", "&emdash;");
-        text.replace("-", "&emdash;");
+        var rep = text.replace(/--/g, "&mdash;");
+        rep.replace("Hello", "&mdash;");
+        return rep;
     }];
 
     return function (scribe) {
         // for now this just exposes a list of
         // commands to do the text cleanup, rather than doing it some
         // weird way with sanitizers
-        var cleanupCommand = new scribe.api.Command("cleanup");
+        var cleanupCommand = new scribe.api.Command();
 
         cleanupCommand.execute = function () {
-            var content = scribe.el.innerText;
-            replacers.forEach(function (f) {
-                f(content);
+            var content = scribe.el.innerHTML;
+
+            var temp = content;
+            reducers.forEach(function (f) {
```

Comment:
At this point shouldn't you be reducing the string rather forEach'ing the transformers?

Or creating a massive partial function?